psydac/feec/multipatch/examples_nc/timedomain_maxwells_testcase.py

```diff
@@ -0,0 +1,251 @@
+import numpy as np
```

Comment:
Can you add a short module docstring? It will be visible in the automatic documentation.